axelrod/tests/unit/test_player.py

```diff
@@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
 
     name = "Player"
```

Comment:
I think the TestPlayer class needs to loop over an ```expected_behaviour``` dictionary and, for each key in that dictionary, check that it exists in the player under test and that the values match. That will cope with existing tests which don't have all the dimensions specified.

But I also think that should be a separate ticket.